package.json

```diff
@@ -20,6 +20,7 @@
     "@angular/platform-browser-dynamic": "~8.1.2",
     "@angular/router": "~8.1.2",
     "@ngrx/store": "^8.1.0",
+    "electron-mock-ipc": "^0.1.3",
```

Comment:
Why do we need the outer package.json? The inner one is for the lib I guess, but is the outer one is a must?